examples/GPU_Examples/SelfCalibrationReconstruction_NonCartesian_gpuNUFFT_SENSE.py

```diff
@@ -0,0 +1,123 @@
+"""
+Non Cartesian Self Calibrating Reconstruction
+====----=====================================
+
+Author: Chaithya G R
+
+In this tutorial we will reconstruct an MRI image from non cartesian kspace
+measurements.
```

Comment:
Maybe specify this a bit more (like what is the specificity of this example compared to `examples/non_cartesian_reconstruction.py`).